cfme/v2v/migration_plans.py

```diff
@@ -417,6 +418,12 @@ def _in_progress():
                             format(plan_name=self.name, visibility=is_plan_visible,
                                    message=new_msg)
                         )
+                        if wait_time:
```

Comment:
This is inconsistent with the rest of the framework in a few ways.

1. Use `wait` kwarg, which we discussed and agreed on for use (see create_view, handle_alert)
2. The default for wait is arguable, but I would use `None`
3. The value passed to `wait` should be seconds, not minutes
4. the docblock should explain the arg

Reply:
Re-reading how this is used, my point #4 is even more important.

Its not a wait, and using `wait` or `wait_time` here is very misleading.

What you're trying to capture here is the _elapsed_ time of the in progress plan.

The name of the method in the first place is completely misleading - as this function does not return a boolean for whether the plan is in progress, it waits for the plan to be _not_ in progress.

The kwarg here should be `elapsed_time`, and the method name should be something like `wait_for_progress`, where by default it waits until the plan is no longer in progress (current behavior) or it waits until a specific elapsed time (what you're trying to add).

Reply:
@mshriver @mnadeem92 : we can also consider creating a small separate method to cancel_migration() that will wait for sometime , cancel migration and then check if the Vm's are migrating anymore and return status if we don't want to modify the current in_progress method